README.md

```diff
@@ -0,0 +1,144 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
```

Comment:
- [ ] Kindly make sure you removed the template logo. Optionally, you can replace it with your own logo if you have one.
![Screenshot from 2022-12-07 03-36-21](https://user-images.githubusercontent.com/65068771/206067005-9edd4703-7af6-4ea9-b08d-48674f630e35.png)